app/config/GmpGlobal.scala

```diff
@@ -75,4 +82,10 @@ trait Hooks extends HttpHooks with HttpAuditing {
 }
 
 trait WSHttp extends HttpGet with WSGet with HttpPut with WSPut with HttpPost with WSPost with HttpDelete with WSDelete with Hooks with AppName
-object WSHttp extends WSHttp
+object WSHttp extends WSHttp {
+  override protected def appNameConfiguration: Configuration = Play.current.configuration
+
+  override protected def actorSystem: ActorSystem = akka.actor.ActorSystem()
```

Comment:
Your current code will create an actorSystem. Please refer to the static dependency of actor system that is in use.
```suggestion
  override protected def actorSystem: ActorSystem = Play.current.actorSystem
```